extensions/pkg/controller/worker/genericactuator/actuator_reconcile.go

```diff
@@ -528,6 +528,15 @@ func getExistingMachineDeployment(existingMachineDeployments *machinev1alpha1.Ma
 	return nil
 }
 
+func isExistingMachineDeploymentWithPositiveReplicaCountPresent(existingMachineDeployments *machinev1alpha1.MachineDeploymentList) int32 {
```

Comment:
The function name suggests that it returns a `bool` but it returns an `int32` 😕

Reply:
changed it to return `bool`